nixos/doc/manual/release-notes/rl-2511.section.md

```diff
@@ -4,6 +4,8 @@
 
 <!-- To avoid merge conflicts, consider adding your item at an arbitrary place in the list instead. -->
 
+- `nixos-rebuild-ng`, a full rewrite of `nixos-rebuild` in Python, is enabled by default from this release. You can disable it by setting [](#opt-system.rebuild.enableNg) to `false` in your configuration if you need, but please report any issues. It is expected that the next major version of NixOS (26.05) will remove `system.rebuild.enableNg` option.
```

Comment:
Grammar nit, should have a "the" before the option:

```suggestion
- `nixos-rebuild-ng`, a full rewrite of `nixos-rebuild` in Python, is enabled by default from this release. You can disable it by setting [](#opt-system.rebuild.enableNg) to `false` in your configuration if you need, but please report any issues. It is expected that the next major version of NixOS (26.05) will remove the `system.rebuild.enableNg` option.
```

(Also: should the non-link option be tagged as `{option}`?)